src/vs/workbench/browser/parts/editor/multiEditorTabsControl.ts

```diff
@@ -2180,6 +2175,7 @@ export class MultiEditorTabsControl extends EditorTabsControl {
 			if (Array.isArray(data)) {
 
 				// Keep the same order when moving / copying editors within the same group
+				// TODO review: this again needs to prevent to work with multiple groups?
```

Comment:
Similar to the editor command, here we only support 1 editor group, no?

Reply:
Yes, here it is only supported from a single group, but that is also the assumption from the data. But I'm fine with making this a guarantee, by having a check here.